block/manager.go

```diff
@@ -819,13 +788,4 @@ func updateState(s *types.State, res *abci.ResponseInitChain) {
 	// We update the last results hash with the empty hash, to conform with RFC-6962.
 	s.LastResultsHash = merkle.HashFromByteSlices(nil)
 
-	if len(res.Validators) > 0 {
```

Comment:
Is `res.Validators > 0` when there was a validator set change on the app side? If so, should we check this to always be empty?

Reply:
we're don't want to allow the app to change the validator set in vA.